src/components/FindWallet/WalletFilterSidebar.tsx

```diff
@@ -0,0 +1,489 @@
+    height: 28px;
+    margin-top: 8px;
+
+    path {
+      fill: white;
+      stroke: white;
+    }
+  }
+`
+
+const OptionDescription = styled.p`
+  font-size: 0.85rem;
+  line-height: 0.8rem;
+  color: ${(props) => props.theme.colors.text200};
+`
+
+const CheckboxGrid = styled.div`
+  display: grid;
+  width: 100%;
+  grid-template-columns: auto auto;
+  margin-top: 14px;
+  gap: 0.5rem;
+`
+
+const CheckboxGridOption = styled.div`
+  display: flex;
+  gap: 0.5rem;
+
+  p {
+    margin: 0;
+  }
+`
+
+const ToggleIcon = styled(Icon)`
+  fill: ${(props) => props.theme.colors.primary};
+`
+
+// Types
+
+const WalletFilterSidebar = ({
+  filters,
+  updateFilterOption,
+  updateFilterOptions,
+}) => {
+  const [filterOptions, setFilterOptions] = useState([
+    {
+      title: "Device",
+      open: true,
+      items: [
+        {
+          title: "Mobile",
+          icon: <Mobile />,
+          description: "Phone or mobile based wallets.",
+          filterKey: undefined,
+          showOptions: false,
+          options: [
+            {
+              name: "Android",
+              filterKey: "android",
+              inputType: "checkbox",
+            },
+            {
+              name: "iOS",
+              filterKey: "ios",
+              inputType: "checkbox",
+            },
+          ],
+        },
+        {
+          title: "Desktop",
+          icon: <Desktop />,
+          description: "Desktop based wallets.",
+          filterKey: undefined,
+          showOptions: false,
+          options: [
+            {
+              name: "Linux",
+              filterKey: "linux",
+              inputType: "checkbox",
+            },
+            {
+              name: "Windows",
+              filterKey: "windows",
+              inputType: "checkbox",
+            },
+            {
+              name: "macOS",
+              filterKey: "macOS",
+              inputType: "checkbox",
+            },
+          ],
+        },
+        {
+          title: "Browser",
+          icon: <Browser />,
+          description: "Browser extension wallets.",
+          filterKey: undefined,
+          showOptions: false,
+          options: [
+            {
+              name: "Firefox",
+              filterKey: "firefox",
+              inputType: "checkbox",
+            },
+            {
+              name: "Chromium",
+              filterKey: "chromium",
+              inputType: "checkbox",
+            },
+          ],
+        },
+        {
+          title: "Hardware",
+          icon: <Hardware />,
+          description: "Hardware baesd wallets.",
+          filterKey: "hardware",
+          showOptions: undefined,
+          options: [],
+        },
+      ],
+    },
+    {
+      title: "Security",
+      open: true,
+      items: [
+        {
+          title: "Open source",
+          icon: <OpenSource />,
+          description:
+            "Wallet code for the wallet is open sourced for users to inspect and edit.",
```

Comment:
This copy could use work (any ideas @minimalsm @wackerow?) - I don't think users editing source code should be the main point here haha. Probably that the source code is transparent/auditable/able to be scrutinized.

Reply:
How about this:

```suggestion
            "Open-source software lets anyone audit the integrity and security of the application.",
```